ledger/tracker.go

```diff
@@ -417,7 +452,7 @@ func (tr *trackerRegistry) commitSyncer(deferredCommits chan *deferredCommitCont
 }
 
 // commitRound commits the given deferredCommitContext via the trackers.
-func (tr *trackerRegistry) commitRound(dcc *deferredCommitContext) {
+func (tr *trackerRegistry) commitRound(dcc *deferredCommitContext) error {
```

Comment:
we made commitRound return an error, but we do not check the error when it is called in commitSyncer